### PR TITLE
Working towards the current master branch

### PR DESCRIPTION
**Pull request by andreadaoud**
I am trying to run elinux on master branch of flutter today. After a whole day's debug, it finally works. This Pull Request can serve as a reference when you adapt it to later flutter releases.

This was worked against flutter commit 37e5bfe240295b7a65fd2d17dde6b2e0fe5a31c2

### CODE REVIEW

**Comment by HidenoriMatsubayashi** [comment]
Thanks.

> I am trying to run elinux on master branch of flutter today. 

We're not following the master branch. I am following the stable branch.

**Comment by andreadaoud** [comment]
Yes, I know, I mean this can serve as a reference when future stable version is released.

**Comment by HidenoriMatsubayashi** [comment]
Okay, thanks.

**Comment by CorvetteCole** [comment]
3.3 stable released, fyi

**Comment by HidenoriMatsubayashi** [comment]
@andreadaoud Can you please update the PR as the [engine artifacts](https://github.com/sony/flutter-embedded-linux/releases/tag/5e9e0e0aa8) is ready?

File: lib/elinux_build_target.dart [hunk 1]
```diff
@@ -87,10 +90,15 @@ abstract class ELinuxAssetBundle extends Target {
...
+    final TargetPlatform tp = buildInfo.targetArch == 'arm64'
+        ? TargetPlatform.linux_arm64
+        : TargetPlatform.linux_x64;
     final Depfile assetDepfile = await copyAssets(
       environment,
       outputDirectory,
-      targetPlatform: null, // corresponds to flutter-tester
+      targetPlatform: tp,
+      buildMode: buildMode,
+      shaderTarget: ShaderTarget.sksl,
```
Comment (HidenoriMatsubayashi) [inline lib/elinux_build_target.dart:101]:
```suggestion
```

**Comment by HidenoriMatsubayashi** [comment]
Needs some updates to fix the warnings.

**Comment by andreadaoud** [comment]
> Needs some updates to fix the warnings.

Could you please do it? I'm working on 3.0.5 now.

**Comment by HidenoriMatsubayashi** [comment]
Can I close this PR once? I'll create a new PR.

**Comment by andreadaoud** [comment]
> Can I close this PR once? I'll create a new PR.

OK

**Comment by HidenoriMatsubayashi** [comment]
Thanks, closing.

**Pull request closed without merging**